packages/svelteui-prism/package.json

```diff
@@ -2,20 +2,6 @@
 	"name": "@svelteuidev/prism",
 	"version": "0.5.0",
 	"sideEffects": false,
-	"scripts": {
-		"dev": "svelte-kit dev",
-		"build": "svelte-kit build",
-		"preview": "svelte-kit preview",
-		"check": "svelte-check --tsconfig ./tsconfig.json",
-		"check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
-		"package": "svelte-kit package",
-		"lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
-		"format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. .",
-		"build:major": "npm --no-git-tag-version version major && npm run package && cd package && npm publish",
-		"build:minor": "npm --no-git-tag-version version minor && npm run package && cd package && npm publish",
-		"build:patch": "npm --no-git-tag-version version patch && npm run package && cd package && npm publish",
-		"build:patch:nopub": "npm --no-git-tag-version version patch && npm run package"
-	},
```

Comment:
I tried now to use aliases in `svelte.config.js` like:

```
vite: {
      test: {
            globals: true,
            environment: 'jsdom'
      },
      resolve: {
            alias: {
                '@svelteuidev/actions/package': '../svelteui-actions/src/lib/index.ts',
                '@svelteuidev/core/package': '../svelteui-core/src/lib/index.ts',
            }
        },
        optimizeDeps: {
            exclude: ['@svelteuidev/actions', '@svelteuidev/core/package'],
        }
}
```

but another error still happens:
![image](https://user-images.githubusercontent.com/25725586/161149415-cd56996c-7e44-40aa-8870-bbb6610054f7.png)

Reply:
The GitHub workflow runs build on all packages, to check that there are no errors when building. Do we not do that for prism?

Reply:
None of the packages need to be "Built." They all use the Package command. I have never run npm build on any of them before. However this is okay for now because I will be completely be redoing the Prism package. So it will not be included in the 5.0 release.

Reply:
You are right, I removed build from the workflow 👍 
Still trying to wrap my head around the concept of monorepo 😅